plugins/build-your-own.rst

```diff
@@ -4,7 +4,7 @@ Building Custom Plugins
 When writing your own Plugin, you need to be aware that the Plugin Client is async first.
 This means that every plugin must be written with Promises. More about this later.
 
-Each plugin must implement the ``Http\Client\Plugin\Plugin`` interface.
+Each plugin must implement the ``Http\Client\Common\Plugin`` interface.
```

Comment:
look at the second example in the symfony doc i linked.

here it could be:

```
.. versionadded: 1.1
    The namespace was adjusted in version 1.1 and the old namespace
    deprecated but will still work until version 2.0. When working with
    version 1.0 you need to use `Http\Client\Plugin\Plugin` instead.
```